broker/lib/jobs/BnRStatusPollerJob.js

```diff
@@ -122,11 +132,116 @@ class BnRStatusPollerJob extends BaseJob {
         }
       })
       .then(operationStatusResponse => operationStatusResponse.operationFinished ?
-        this.doPostFinishOperation(operationStatusResponse, operationName, instanceInfo) :
-        Promise.resolve(operationStatusResponse)
+        this.doPostFinishOperation(operationStatusResponse, operationName, instanceInfo) : operationStatusResponse
       );
   }
+
+  static checkOperationCompletionStatus20(job_data) {
+    logger.info('Checking Operation Completion Status for :', job_data);
+    const operationName = job_data.operation;
+    const instanceInfo = job_data.operation_details;
+    const instance_guid = instanceInfo.instance_guid;
+    const backup_guid = instanceInfo.backup_guid;
+    const deployment = instanceInfo.deployment;
+    const plan = catalog.getPlan(instanceInfo.plan_id);
+    return Promise.try(() => {
+        if (operationName === CONST.OPERATION_TYPE.BACKUP) {
+          return lockManager.lock(instance_guid, {
```

Comment:
I thought backup manager and backup jobs should not care about locks, so why is lock being retrieved here?